### PR TITLE
Put doctrine-migrations into including project's vendor/bin directory

### PR DESCRIPTION
**Pull request by dotEvan**
Enables a project to include doctrine/migrations and then to run any migration commands by running

```
./vendor/bin/doctrine-migrations
```

from the main project's directory.

### CODE REVIEW

**Comment by mikeSimonson** [comment]
Thanks @dotEvan .

Don't we need to set as executable only the doctrine-migrations file and not the php one ?

File: composer.json [hunk 1]
```diff
@@ -35,5 +35,9 @@
         "branch-alias": {
             "dev-master": "1.0.x-dev"
         }
-    }
+    },
+    "bin": [
+        "bin/doctrine-migrations",
+        "bin/doctrine-migrations.php"
```
Comment (stof) [inline composer.json]: `` bin/doctrine-migrations.php`` should not be there. Only the actual bin is needed

**Comment by stof** [comment]
@mikeSimonson all files listed in the ``bin`` section of composer must be executable (otherwise Composer will make them executable and then the next update will tell you that the package has some changes).
But the mistake is listing the PHP file in this section in the first place

File: composer.json (hunk 1 above)
Comment (dotEvan) [inline composer.json]: @stof To be honest, I only use the version without the .php. However, I was being consistent with the way this was done in https://github.com/doctrine/doctrine2/blob/master/composer.json

I didn't want to assume that people didn't also want to be able to do
```
php ./vendor/bin/doctrine-migrations.php
```
if they wanted to.

**Comment by n-couet** [comment]
+1

File: composer.json (hunk 1 above)
Comment (stof) [inline composer.json]: And this is a mistake in the ORM (it was reverted in DBAL but I missed the fact that the ORM included it too)

**Comment by mikeSimonson** [comment]
@dotEvan Can you make that change in your PR ?

Revert the change of permission on bin/doctrine-migrations.php and remove bin/doctrine-migrations.php from the bin section of the composer.json ?

**Comment by dotEvan** [comment]
@mikeSimonson Done.

**Comment by mikeSimonson** [comment]
@dotEvan Thanks

**Pull request merged**